packages/suite/src/components/suite/Settings/components/ActionColumn.tsx

```diff
@@ -19,7 +19,7 @@ export const ActionColumn = styled.div`
 
 const InputWrapper = styled.div`
     margin: 4px 0 4px 4px;
-    width: 170px;
+    width: 210px;
```

Comment:
It is really needed? 16 chars can fit into 170px

Reply:
That is true but error messages will be splitted into two lines. I think we don't want that.

Reply:
I discussed this with @Hermez-cz and basically he said that design is not a priority now since we're going to have new design in the upcoming months

Reply:
Can you post here a screen shots of both solutions?

Reply:
`210px`:
<img width="231" alt="Screenshot 2023-08-03 at 3 51 39 PM" src="https://github.com/trezor/trezor-suite/assets/66002635/ea1d8cbe-9c54-4679-bb20-030f7b0643e2">
<img width="225" alt="Screenshot 2023-08-03 at 3 50 23 PM" src="https://github.com/trezor/trezor-suite/assets/66002635/8fdd7f3f-5de8-46ab-9dd7-405ea2ffba75">


`170px`:
<img width="197" alt="Screenshot 2023-08-03 at 3 52 35 PM" src="https://github.com/trezor/trezor-suite/assets/66002635/1ca9012a-3031-49c8-8c80-a5c645c9138e">
<img width="203" alt="Screenshot 2023-08-03 at 3 52 23 PM" src="https://github.com/trezor/trezor-suite/assets/66002635/cb406571-4e03-41a7-b706-c006e0dd70be">

Reply:
Thanks! I would use the shorter one as displaying the message seems corner-case-ish.